### PR TITLE
Full scalar support

### PR DESCRIPTION
**Pull request by MacondoExpress**
# Description

As title.

### CODE REVIEW

File: packages/graphql/src/api-v6/queryIRFactory/resolve-tree-parser/ResolveTreeParser.ts [hunk 1]
```diff
@@ -75,8 +82,54 @@ export abstract class ResolveTreeParser<T extends ConcreteEntity | Relationship>
...
+                        longitude: resolveTreeToLeafField(longitude),
+                        latitude: resolveTreeToLeafField(latitude),
+                        height: resolveTreeToLeafField(height),
+                        crs: resolveTreeToLeafField(crs),
+                        srid: resolveTreeToLeafField(srid),
+                    },
+                };
+
+                return pointField;
+            }
```
Comment (angrykoala) [inline packages/graphql/src/api-v6/queryIRFactory/resolve-tree-parser/ResolveTreeParser.ts]: Could we move this to a separate function?

File: packages/graphql/src/api-v6/schema-generation/schema-types/StaticSchemaTypes.ts [hunk 2]
```diff
@@ -181,32 +283,37 @@ class StaticFilterTypes {
...
+                    equals: toGraphQLList(toGraphQLNonNull(GraphQLLocalTime)),
                 },
             };
         });
     }
 
+    public getBooleanListWhere(nullable: boolean): InputTypeComposer {
+        //  TODO: verify correctness of this
+        return this.getStringListWhere(nullable);
+    }
```
Comment (angrykoala) [inline packages/graphql/src/api-v6/schema-generation/schema-types/StaticSchemaTypes.ts]: I'm not sure these types are the same. Boolean does not have the same types as string

File: packages/graphql/src/schema-model/attribute/model-adapters/AttributeAdapter.test.ts [hunk 3]
```diff
@@ -25,10 +25,12 @@ import {
     GraphQLBuiltInScalarType,
     InterfaceType,
     ListType,
-    Neo4jCartesianPointType,
+    //  Neo4jCartesianPointType,
```
Comment (angrykoala) [inline packages/graphql/src/schema-model/attribute/model-adapters/AttributeAdapter.test.ts]: Can we remove these?

File: packages/graphql/src/schema-model/parser/parse-attribute.ts [hunk 4]
```diff
@@ -100,9 +99,11 @@ function parseTypeNode(
             if (isScalarType(typeNode.name.value)) {
                 return new ScalarType(typeNode.name.value, isRequired);
             } else if (isPoint(typeNode.name.value)) {
-                return new Neo4jPointType(isRequired);
+                return new Neo4jSpatialType(typeNode.name.value, isRequired);
+                // return new Neo4jPointType(isRequired);
```
Comment (angrykoala) [inline packages/graphql/src/schema-model/parser/parse-attribute.ts]:
```suggestion
```

File: packages/graphql/tests/api-v6/integration/filters/types/cartesian-point/cartesian-point-2d-equals.int.test.ts [hunk 5]
```diff
@@ -0,0 +1,156 @@
...
+            { London, Rome }
+        );
+        await testHelper.initNeo4jGraphQL({ typeDefs });
+    });
+
+    afterEach(async () => {
+        await testHelper.close();
+    });
+
+    test("wgs-84-2d point filter by EQ", async () => {
```
Comment (angrykoala) [inline packages/graphql/tests/api-v6/integration/filters/types/cartesian-point/cartesian-point-2d-equals.int.test.ts:54]: What does `wgs-84-2d` stands for?

**Review by angrykoala — APPROVED** [review]

File: packages/graphql/tests/api-v6/integration/filters/types/cartesian-point/cartesian-point-2d-equals.int.test.ts (hunk 5 above)
Comment (MacondoExpress) [reply packages/graphql/tests/api-v6/integration/filters/types/cartesian-point/cartesian-point-2d-equals.int.test.ts:54]: Is the standard used by for the Spatial Types: https://en.wikipedia.org/wiki/World_Geodetic_System#WGS84 
https://neo4j.com/docs/cypher-manual/current/values-and-types/spatial/#spatial-values-crs
!

File: packages/graphql/src/schema-model/parser/parse-attribute.ts (hunk 4 above)
Comment (MacondoExpress) [reply packages/graphql/src/schema-model/parser/parse-attribute.ts]: Good catch!

**Pull request merged**